src/styles/card.css

```
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  margin: auto;
+
+  height: 134px;
+}
+
+.card__product-name {
```

Do you really need a product word in this class name?
```suggestion
.card__name {
```